app/src/main/res/values/strings.xml

```diff
@@ -376,6 +376,11 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
     <string name="expand_collapse">Expand / Collapse</string>
     <string name="essential">Essential</string>
     <string name="append_human_readable_uuid_to_new_tasks">Append human readable UUID to new tasks</string>
+    <string name="document_password">Password for file encryption.</string>
+    <string name="no_password_found_warning"><![CDATA[No password found, can\'t en-/decrypt.\nGo to \'Settings -> General -> Password\' to set a password.]]></string>
+    <string name="encrypt_file_content">encrypt file content</string>
+    <string name="failed_to_encrypt_document">Couldn\'t encrypt_message. The document wasn\'t stored.</string>
+    <string name="failed_to_decrypt_document">Couldn\'t decrypt message. Wrong password?</string>
```

Comment:
could_not_decrypt_file_content_wrong_password_or_is_the_file_maybe_not_encrypted
Could not dencrypt file content. Did you use a wrong password or is the file maybe not encrypted?

Reply:
Long texts in teasers are difficult to read, but changed it.